extensions/deploy_replace_undeploy/standard/sections/clause_7_apppkg.adoc

```diff
@@ -45,12 +45,8 @@ include::../requirements/ogcapppkg/REQ_process-description.adoc[]
 ==== executionUnit property
 
 The execution unit is the `code` that an implementation of this extensions needs
-to execute whenever the deployed process is executed.  The execution unit can be
-either specified in-line (e.g. CWL script) or by reference (e.g. docker image reference).
-
-include::../requirements/ogcapppkg/REQ_execution-unit-docker.adoc[]
-
```

Comment:
Is this removed on purpose? 
If the implementation does not use CWL, it will not have a reference to the alternative anymore?
I wouldn't mind because I believe CWL is more descriptive, but just to be sure this was intentional.

Reply:
Yes, this has been removed on purpose. This was related to the TBD sections that were removed.

TBD sections removed are listed below:
 * [Providing value hints for process inputs](https://github.com/opengeospatial/ogcapi-processes/blob/38ce0b5b84f6bfd0eb0cdb593c290ccca64a8b9b/extensions/deploy_replace_undeploy/standard/sections/clause_7_apppkg.adoc#providing-value-hints-for-process-inputs)
 * [Binding the process description to the invocation of the execution unit](https://github.com/opengeospatial/ogcapi-processes/blob/38ce0b5b84f6bfd0eb0cdb593c290ccca64a8b9b/extensions/deploy_replace_undeploy/standard/sections/clause_7_apppkg.adoc#binding-the-process-description-to-the-invocation-of-the-execution-unit)
 * [deploymentProfile property](https://github.com/opengeospatial/ogcapi-processes/blob/38ce0b5b84f6bfd0eb0cdb593c290ccca64a8b9b/extensions/deploy_replace_undeploy/standard/sections/clause_7_apppkg.adoc#deploymentprofile-property)

I understand all these TBD sections are related to each other and with the requirements. In consequence, all were removed.

If we can find consensus on the first version, we can add more to the OGC Application Package later (in a new version).